config/default/field.field.node.judgment.field_files.yml

```diff
@@ -39,7 +39,7 @@ default_value_callback: ''
 settings:
   file_directory: '[date:custom:Y]-[date:custom:m]'
   file_extensions: 'txt doc docx xls pdf ppt pps odt ods odp rtf'
-  max_filesize: '10 MB'
+  max_filesize: '50 MB'
```

Comment:
This file needs to be deleted.

Reply:
This is to change the max_filesize for the the judgments download files to 50mb.

Reply:
Whoops, that's fine, thanks.